src/main/java/net/neoforged/moddevgradle/internal/jarjar/JarJarArtifacts.java

```diff
@@ -234,16 +237,31 @@ private static String validateVersionRange(String range, ModuleComponentSelector
         }
 
         if (!data.hasRestrictions()) {
-            throw new GradleException(errorPrefix + "no restrictions");
-        } else if (data.getRecommendedVersion() != null) {
-            throw new GradleException(errorPrefix + "recommended versions are unsupported");
-        } else if (range.endsWith("+") || range.startsWith("latest.")) {
+            // If there was no restriction, just use the current version as the lower bound
+            return makeOpenRange(variant);
```

Comment:
This is a bit weird, we already have that as a fallback in `collectFromComponent`.